web-server/src/components/GithubButton.tsx

```diff
@@ -0,0 +1,59 @@
+import StarBorderOutlined from '@mui/icons-material/StarBorderOutlined';
+import {
+  Button,
+  ButtonProps,
+  Divider,
+  Typography,
+  useTheme
+} from '@mui/material';
+import { FC, useMemo } from 'react';
+
+import { useSelector } from '@/store';
+
+const githubRepoUrl = `https://github.com/middlewarehq/middleware`;
+
+const formatStarCount = (stars: number) => {
+  if (isNaN(stars)) return '';
+  if (stars < 1000) return `${stars}`;
+
+  let digit = `${(stars / 1000).toFixed(1)}`;
+  if (digit.endsWith('0')) digit = digit.slice(0, -2);
+
+  return `${digit}k`;
+};
```

Comment:
calling a function into a `useMemo` requires it to be wrapped in a `useCallback`

Reply:
I don't think it is needed as I declared the function globally 🤔

Reply:
That is correct. I think @e-for-eshaan might have missed and thought this was declared inside the component, since I almost thought that myself before seeing your comment. :D